rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -3163,7 +3163,7 @@ class MenuController extends ControllerBase implements ApplicationContextAware{
         }
 
 
-        def list = ScheduledExecution.findAllByServerNodeUUIDAndScheduled(uuid,true)
+        def list = jobSchedulesService.getAllScheduled(uuid, null)
```

Comment:
I don't think you need to add in null as the second parameter since the stub shows
getAllScheduled(String serverUUID = null, String project = null)